airflow/sensors/base.py

```diff
@@ -122,6 +135,7 @@ def __init__(
         self.timeout = timeout
         self.mode = mode
         self.exponential_backoff = exponential_backoff
+        self.max_wait = max_wait if max_wait is None else coerce_timedelta(max_wait, key="max_retry_delay")
```

Comment:
i would put "if wait is none" logic in coerce timedelta so that ... if it is none, it will return none, wdyt?

Reply:
yes it would be cleaner